test/spec/Feature/Query/InsertSpec.hs

```diff
@@ -572,6 +572,15 @@ spec actualPgVersion = do
               , matchHeaders = ["Preference-Applied" <:> "missing=default, return=representation"]
               }
 
+        it "inserts a COLUMN default before a DOMAIN default with missing=default" $
+          request methodPost "/evil_friends_with_column_default?columns=id,name" [("Prefer", "return=representation"), ("Prefer", "missing=default")]
```

Comment:
Are there tests that ensure a DOMAIN default will be applied in other cases?

Reply:
Yes, it is present just above this new test.